src/win/fs.c

```diff
@@ -1403,13 +1404,18 @@ static void fs__fchmod(uv_fs_t* req) {
 }
 
 
-INLINE static int fs__utime_handle(HANDLE handle, double atime, double mtime) {
-  FILETIME filetime_a, filetime_m;
+INLINE static int fs__utime_handle(HANDLE handle, double btime, double atime, double mtime) {
+  FILETIME filetime_b, filetime_a, filetime_m;
+  int btime_provided = !isnan(btime);
+
+  if (btime_provided) {
+    TIME_T_TO_FILETIME((time_t) btime, &filetime_b);
+  }
 
   TIME_T_TO_FILETIME((time_t) atime, &filetime_a);
   TIME_T_TO_FILETIME((time_t) mtime, &filetime_m);
 
-  if (!SetFileTime(handle, NULL, &filetime_a, &filetime_m)) {
+  if (!SetFileTime(handle, btime_provided ? &filetime_b : NULL, &filetime_a, &filetime_m)) {
```

Comment:
Sure.  I noticed that this file has a number of formatting issues and wasn't sure which to honor.